### PR TITLE
Merged mining introduction changes

### PR DESCRIPTION
**Pull request by anselld**
Formatting complete. [27] and [28] were not referred to in the report - suggest deleting.

### CODE REVIEW

File: src/merged-mining/merged-mining-scene/MergedMiningIntroduction.md [hunk 1]
```diff
@@ -102,122 +125,241 @@ A miner can use a single Parent to perform merged mining on multiple Auxiliary b
...
+- Tari tokens are envisaged to be merged mined with Monero [[13]], as such the Monero blockchain security is important to the Tari blockchain. 
+- Monero recently (2018-04-06) introduced a hard fork with upgraded PoW algorithm CryptoNight v7 at block height 1,546,000 to maintain their Application Specific Integrated Circuit (ASIC) resistance and hence guard against 51% attacks. The Monero team proposes changes to their PoW every scheduled fork (i.e. every 6 months) going forward. ([[14]], [[15]])
+- An interesting question arises what needs to happen to the Tari blockchain if the Monero blockchain is hard forked. Since the CryptoNight v7 hard fork the network hash rate for Monero hovers around ~500 MH/s, whereas in the two months immediately prior it was ~1,000 MH/s [[20]](https://chainradar.com/xmr/chart). Thus 50% of the hash power can be ascribed to ASICS and botnet miners.
+
+![MoneroHashRate](./sources/MoneroHashRate.png)
+
+NiceHash statistics for CryptoNight v7 [[21]](https://www.nicehash.com/algorithm/cryptonightv7) shows a lag of 2 days for ~ 100,600 miners to get up to speed with providing the new hashing power after the Monero hard fork.
```
Comment (hansieodendaal) [inline src/merged-mining/merged-mining-scene/MergedMiningIntroduction.md]:
```suggestion
NiceHash statistics for CryptoNight v7 [[21]] shows a lag of 2 days for ~ 100,600 miners to get up to speed with providing the new hashing power after the Monero hard fork.
```

File: src/merged-mining/merged-mining-scene/MergedMiningIntroduction.md [hunk 2]
```diff
@@ -102,122 +125,241 @@ A miner can use a single Parent to perform merged mining on multiple Auxiliary b
...
-[23] Myriad: A Coin For Everyone, http://myriadcoin.org, Accessed: 2018-06-01.
+[26]: https://github.com/ethereum/wiki/wiki/Problems
+"Problems - Consensus - 8. Proof of Stake"
 
-[24] Merged Mining: Curse or Cure?,  https://eprint.iacr.org/2017/791.pdf, Judmayer et. al., Data Privacy Management, Cryptocurrencies and block chain Technology: ESORICS 2017 International Workshops, DPM 2017 and CBT 2017, Oslo, Norway, September 14-15, 2017, Proceedings (pp.316-333).
+## Contributors
 
-[25] Merged Mining: Analysis of Effects and Implications,  http://repositum.tuwien.ac.at/obvutwhs/download/pdf/2315652, Zamyatin Alexei, MSc Thesis, Faculty of Informatics at the Technische Universität Wien.
+- [https://github.com/hansieodendaal]
+- [https://github.com/anselld]
```
Comment (hansieodendaal) [inline src/merged-mining/merged-mining-scene/MergedMiningIntroduction.md]:
```suggestion
- [https://github.com/CjS77](https://github.com/CjS77)
- [https://github.com/anselld]
```

File: src/merged-mining/merged-mining-scene/MergedMiningIntroduction.md [hunk 3]
```diff
@@ -1,80 +1,103 @@
 # Merged Mining Introduction
 
-## What is merged mining? 
+[What is Merged Mining?](#what-is-merged-mining)
 
-Merged mining is the act of using work done on another block chain (the Parent) on one or more Auxiliary block chains and to accept it as valid on its own chain, using Auxiliary Proof-of-Work (AuxPoW), which is the relationship between two block chains for one to trust the other's work as their own. The Parent block chain does not need to be aware of the AuxPoW logic as blocks submitted to it are still valid blocks. [[1]](https://en.bitcoin.it/wiki/Merged_mining_specification)
+[Merged Mining with Multiple Auxiliary Chains](#merged-mining-with-multiple-auxiliary-chains)
 
-As an example the structure of merged mined blocks in Namecoin and Bitcoin is shown below. [[25]](http://repositum.tuwien.ac.at/obvutwhs/download/pdf/2315652)
+[Merged Mining - Interesting Facts, Case Studies](#merged-mining-interesting-facts-case-studies)
```
Comment (hansieodendaal) [inline src/merged-mining/merged-mining-scene/MergedMiningIntroduction.md]:
```suggestion
[Merged Mining - Interesting Facts, Case Studies](#merged-mining---interesting-facts-case-studies)
```

File: src/merged-mining/merged-mining-scene/MergedMiningIntroduction.md (hunk 2 above)
Comment (hansieodendaal) [reply src/merged-mining/merged-mining-scene/MergedMiningIntroduction.md]: Please also change line 363 from `- [https://github.com/hansieodendaal]` to: `- [https://github.com/hansieodendaal](https://github.com/hansieodendaal)`

File: src/merged-mining/merged-mining-scene/MergedMiningIntroduction.md (hunk 1 above)
Comment (anselld) [reply src/merged-mining/merged-mining-scene/MergedMiningIntroduction.md]: Thanks - I missed that one.

Comment (hansieodendaal) [reply src/merged-mining/merged-mining-scene/MergedMiningIntroduction.md]: Hi Denise, seems you missed to apply this suggested change.

**Comment by hansieodendaal** [comment]
Hi @anselld, thanks, please go ahead with editing, just one missed comment to take car of as well.

**Comment by anselld** [comment]
Thanks, Hansie. Fixed missed comment and have started editing.

**Comment by anselld** [comment]
Completed editing. Looks good. One concern: in the tables, I have inserted commas as thousands separators in the 'Blocks count' row. Are the full stops in the preceding 'Hash rate' row correct?

**Pull request merged**